06-building-a-program-start-to-finish/solving-the-problem/output/InvertedIndex.js

```diff
@@ -0,0 +1,131 @@
+/**
+ * InvertedIndex class
+ * @class
+ */
+class InvertedIndex {
+  /**
+   * class constructor
+   * @constructor
+   */
+  constructor() {
+    this.files = [];
+  }
+
+  /**
+   * Validates file content
+   * @function
+   * @param {Array} fileContent content of uploaded file
+   * @return {Boolean} true if a valid file was uploaded, false otherwise
+   */
+  static validateFile(fileContent) {
+    if ((fileContent instanceof Array) && fileContent.length > 0) {
+      let valid = true;
+      fileContent.forEach((book) => {
+        if (!(book.title && book.text)) {
+          valid = false;
+          return;
+        }
+        if (book.text.length < 1) valid = false;
```

Comment:
Yes, that is ideal but I noticed something interesting:
`return` in a forEach() loop does not exit the function, it only skips the rest of the code in the forEach and continues the loop (essentially, it is serving as `continue`)